test/convexhull.jl

```diff
@@ -0,0 +1,7 @@
+@testset "upperhull" begin
+  
+  pset1=PointSet((0.,0.),(1.,1.),(1.5,1.0))
```

Comment:
Please use the same format of the other tests with `P2` which will be changed to single and double floating point precision.